src/main/java/org/threeten/extra/OffsetDate.java

```diff
@@ -1278,40 +1319,15 @@ public String toString() {
      * Outputs this date as a {@code String} using the formatter.
      * <p>
      * This date will be passed to the formatter
-     * {@link DateTimeFormatter#print(TemporalAccessor) print method}.
+     * {@link DateTimeFormatter#format(TemporalAccessor) print method}.
      *
      * @param formatter  the formatter to use, not null
      * @return the formatted date string, not null
      * @throws DateTimeException if an error occurs during printing
      */
     public String toString(DateTimeFormatter formatter) {
         Objects.requireNonNull(formatter, "formatter");
-        return formatter.print(this);
-    }
-
-    //-----------------------------------------------------------------------
-    private Object writeReplace() {
-        return new Ser(Ser.OFFSET_DATE_TYPE, this);
-    }
-
-    /**
-     * Defend against malicious streams.
-     * @return never
-     * @throws InvalidObjectException always
-     */
-    private Object readResolve() throws ObjectStreamException {
-        throw new InvalidObjectException("Deserialization via serialization delegate");
-    }
-
-    void writeExternal(DataOutput out) throws IOException {
-        date.writeExternal(out);
-        offset.writeExternal(out);
-    }
-
-    static OffsetDate readExternal(DataInput in) throws IOException {
-        LocalDate date = LocalDate.readExternal(in);
-        ZoneOffset offset = ZoneOffset.readExternal(in);
-        return OffsetDate.of(date, offset);
```

Comment:
I did not know what to do with these helper methods for implementing externalizable, so I removed them for now.